.github/workflows/staging-odfe-cli.yml

```diff
@@ -0,0 +1,64 @@
+name: Staging ODFE CLI
+
+on:
+  schedule:
+    - cron: '0 12 * * *'
+  repository_dispatch:
+    types: [staging-odfe-cli]
+
+jobs:
+  Test-ODFE-CLI:
+    runs-on: ubuntu-18.04
+    name: Test-ODFE-CLI
+    strategy:
+      matrix:
+        go-version: [1.14]
+    steps:
+      - name: Set up AWS Cred
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_STAGING_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_STAGING_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
```

Comment:
Is matrix really required? Any possibilities that there are going to more than one go versions in future?

Reply:
They will in future, so this gives more flexibility in running the IT.